modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -760,6 +761,109 @@ public void testAddObjectEntry() throws Exception {
 				_simpleDTOConverterContext, _objectDefinition1,
 				parentObjectEntry1.getId()));
 
+		// Aggregation field without filters based on Many to Many relationship
```

Comment:
@pedro-oliveira446 rename: `// Aggregation field without filters (many to many self relationship)`